beacon-chain/core/blocks/block_operations_test.go

```diff
@@ -44,6 +45,21 @@ func setupInitialDeposits(t *testing.T, numDeposits int) ([]*pb.Deposit, []*bls.
 	return deposits, privKeys
 }
 
+func createAggregateSignature(t *testing.T, beaconState *pb.BeaconState, att *pb.Attestation, privKey *bls.SecretKey) []byte {
+	attestationDataHash, err := hashutil.HashProto(&pb.AttestationDataAndCustodyBit{
+		Data:       att.Data,
+		CustodyBit: true,
+	})
+	if err != nil {
+		t.Errorf("could not hash attestation data: %v", err)
```

Comment:
if you cant hash the attestation data , your whole test will fail , no ? it should be a fatal in that case